packages/roosterjs-editor-core/lib/coreApi/getPendableFormatState.ts

```diff
@@ -15,19 +15,19 @@ import {
  */
 export const getPendableFormatState: GetPendableFormatState = (
     core: EditorCore,
-    forceGetStateFromDOM: boolean
+    forceGetStateFromDOM: boolean = false
```

Comment:
Do not mark parameter of core API to optional. We expect all parameters of core API to be explicitly passed.